xdsl/irdl_mlir_printer.py

```diff
@@ -78,7 +78,7 @@ def print_attr_constraint(self, f):
     def print_parameters_definition(self, param_op: ParametersOp):
         self._print(f"      {ParametersOp.name}", end="(")
 
-        for param in param_op.constraints.data:
+        for param in param_op.params.data:
```

Comment:
This looks a bit ugly... like param in params in params....
Any ideas to improve this?

Reply:
Hmm not really, though this file will be removed relatively soon, so that shouldn't be much of a problem.
This commit is moving forward to print/parse IRDL from xDSL directly, without going through that file